src/lauxlib.js

```diff
@@ -399,6 +400,124 @@ const luaL_newlib = function(L, l) {
     luaL_setfuncs(L, l, 0);
 };
 
+// Only with Node
+if (typeof window === "undefined") {
```

Comment:
Actually, probably better to do a feature check for what we want. i.e. `if (typeof require === "function") { try { fs = require("fs"); } ......`